site/content/en/blog/releases/1.29.0.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "1.29.0 - Kubernetes 1.24.0, SDKGracefulTermination feature in beta, New PodHostname feature"
+linktitle: "1.29.0"
+date: "2023-01-17"
+---
+
+This is the 1.29.0 release of Agones.
+
+We are bringing a huge release to start 2023!
+* Update Agones to support for Kubernetes 1.24.0
+* SDKGracefulTermination feature has now graduated to beta. So by default, the sidecar container will not shutdown immeadiately on TERM signal, but instead wait for Shutdown state to occur.
+* A new PodHostname feature is now available, if you need to able to access a GameServer Pod within a cluster by a DNS name.
+* Agones release images have all been moved to [Google Artifact Registry](https://cloud.google.com/artifact-registry)
```

Comment:
nit: Can you append "(prior releases will still be accessible in their previous location)"